tests/test_lab_serve.py

```diff
@@ -335,3 +335,19 @@ def test_warn_for_unsupported_backend_param(param, expected_call_count):
             mock_warning.assert_called_with(
                 f"Option '--{param.replace('_','-')}' not supported by the backend."
             )
+
+
+def test_host_port(cli_runner: CliRunner):
+    args = common.vllm_setup_test(
+        cli_runner,
+        [
+            "--config=DEFAULT",
+            "model",
+            "serve",
+            "--model-path=foo",
+            "--host-port",
+            "192.168.1.1:8000",
+        ],
+    )
+    assert "192.168.1.1" in args
```

Comment:
I think the config tests technically may cover this, but we should add an explicit test in this same case which validates that when host-port isn't provided then it uses the default value.